_maps/slimeguyspaydayexpress/slimeguyspaydayexpress.yaml

```diff
@@ -0,0 +1,182 @@
+---
+name:
+  en: Slime Guy's Payday Express
+desc:
+  en: All aboard the Mushroom Kingdom's high speed, non-stop & squishy railway system. One ticket takes you straight to your next payday destination! Well...that is as long as none of the passengers step on the switches...
+ruleSet: Standard
+theme: DragonQuest
+initialCash: 2500
+targetAmount: 20000
+baseSalary: 350
+salaryIncrement: 250
+maxDiceRoll: 6
+frbFile1: slimeguyspaydayexpress1s
+frbFile2: slimeguyspaydayexpress2s
+frbFile3: slimeguyspaydayexpress3s
+frbFile4: slimeguyspaydayexpress4s
+background: bg109
+mapIcon: paydayexpress
+music:
+  download: 
+   - https://nikkums.io/cswt/BGM/slimeguyspaydayexpress.music.zip
+   - https://drive.google.com/file/d/16y8nk0E1Xjh2fH_d0xAb1zAc97A6qxI6/view?usp=drive_link

```

Comment:
This one should be `https://drive.google.com/uc?export=download&confirm=1&id=16y8nk0E1Xjh2fH_d0xAb1zAc97A6qxI6`